### PR TITLE
Add Yii PHP framework

### PR DESCRIPTION
**Pull request by gekh**
PHP framework Yii in PHP folder.

Yii version: 2.0.32

Issue: https://github.com/the-benchmarker/web-frameworks/issues/634

To start the server go to "web" folder and run
php -S localhost:8000

### CODE REVIEW

File: php/yii/web/index.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+// comment out the following two lines when deployed to production
+defined('YII_DEBUG') or define('YII_DEBUG', false);
+defined('YII_ENV') or define('YII_ENV', 'prod');
+defined('YII_ENABLE_ERROR_HANDLER') or define('YII_ENABLE_ERROR_HANDLER', false);
```
Comment (samdark) [inline php/yii/web/index.php]: Should not be disabled. Usually production runs with Yii's error handling.

```suggestion
```

File: php/yii/controllers/SiteController.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace app\controllers;
+
+use yii\web\Controller;
+
+class SiteController extends Controller
+{
+    public $layout = false;
```
Comment (samdark) [inline php/yii/controllers/SiteController.php]: I see no need to disable it. It should not affect anything significantly.

File: php/yii/controllers/SiteController.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    public $enableCsrfValidation = false;
```
Comment (samdark) [inline php/yii/controllers/SiteController.php]: I see no need to disable it. It should not affect anything significantly.

File: php/yii/controllers/UserController.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace app\controllers;
+
+use yii\web\Controller;
+
+class UserController extends Controller
+{
+    public $layout = false;
```
Comment (samdark) [inline php/yii/controllers/UserController.php]: I see no need to disable it. It should not affect anything significantly.

File: php/yii/controllers/UserController.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    public $enableCsrfValidation = false;
```
Comment (samdark) [inline php/yii/controllers/UserController.php]: I see no need to disable it. It should not affect anything significantly.

**Comment by gekh** [comment]
> Thanks for you awesome contribution 🎉
> 
> As you have made it from **master**, it is not _writable_ for any of us, could you please invite me as contributor so as I can participate ?

Oh, sorry. I didn't know about that. I've added you (@waghanza) and @samdark as collaborators. I hope now you would be able to participate.

File: php/yii/controllers/SiteController.php (hunk 3 above)
Comment (gekh) [reply php/yii/controllers/SiteController.php]: It brokes POST request:
POST /user, SHOULD return a successful status code and an empty string

File: php/yii/controllers/UserController.php (hunk 5 above)
Comment (gekh) [reply php/yii/controllers/UserController.php]: It brokes POST request:
POST /user, SHOULD return a successful status code and an empty string

Comment (samdark) [reply php/yii/controllers/UserController.php]: Ah, yes. It is correct.

**Pull request merged**